workers/send_sms.js

```diff
@@ -0,0 +1,21 @@
+var twilio = require("twilio");
+
+var twilioClient = twilio(process.env.TWILIO_SID, process.env.TWILIO_AUTH_TOKEN);
```

Comment:
This should be passed in via config (like google spreadsheet worker) not  captured from the environment here